src/mixins/common-mixin.ts

```diff
@@ -112,4 +115,14 @@ export default class CommonMixin extends Vue {
         return ''
     }
   }
+
+  /**
+   * Method to compare the conditional entity to the entityType defined from the Store.
+   *
+   * @param entity The entity type of the component.
+   * @return boolean A boolean indicating if the entityType given matches the entityType assigned to the component.
+   */
+  entityFilter (entityType: EntityTypes): boolean {
```

Comment:
For consistency, I suggest moving this to store getters and renaming it to `isEntityType` or something...

Reply:
(See isCoop/isBComp.)

Reply:
Actually, maybe `isBen` or `isCr`???

Reply:
Well, that is one approach. It would have to be a getter specific to the type we wanted to check as the getters don't take parameters. 
Long term, we will support many more types, do we want that many getters?

Reply:
I'm impartial but it's worth talking about :)

Reply:
You're right. I think atm we don't know enough to make a long term decision, so just leave it :)